.rubocop.yml

```diff
@@ -0,0 +1,80 @@
+inherit_from:
+  - ../.rubocop.yml
+
+Bundler/GemFilename:
+  Enabled: false
+
+Homebrew/MoveToExtendOS:
+  Enabled: true
+  Exclude:
+    - "{extend,test,requirements}/**/*"
+    - "os.rb"
```

Comment:
A lot of this is intentionally Homebrew/brew like this cop (we don't want to block if OS.mac? in taps). Is this the correct file?

Reply:
Yup, not correct, see https://github.com/Homebrew/.github/pull/111 to fix.